libs/pages/elewa/brands/src/lib/brands.routing.ts

```diff
@@ -14,4 +14,5 @@ export const ELEWA_BRANDS_ROUTES: Route[] = [
   imports: [RouterModule.forChild(ELEWA_BRANDS_ROUTES)],
   exports: [RouterModule]
 })
-export class BrandsRoutingModule { }
\ No newline at end of file
+export class BrandsRoutingModule { }
+
```

Comment:
This change is not necessary

Reply:
It was an issue to create the  brands lib component